Bottom: 31bc4e539c2e2d95b81b9c9723cca51b2d9c6587
Top:    d82222c2c78634352bb339ba72f39de5b2846159
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-26 01:10:07 +0100

Refresh of draw-c-split-gathering-routine

---

diff --git a/src/draw.c b/src/draw.c
index 25e446d..9051686 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -399,7 +399,7 @@ draw_element_pins_and_pads (ElementType *element)
   PAD_LOOP (element);
   {
     if (doing_pinout || doing_assy || FRONT (pad) || PCB->InvisibleObjectsOn)
-      DrawPad (pad);
+      draw_pad (pad);
   }
   END_LOOP;
   PIN_LOOP (element);
@@ -1556,7 +1556,6 @@ DrawElementPackage (ElementTypePtr Element)
 void
 DrawElementPinsAndPads (ElementTypePtr Element)
 {
-<<<<<<< current
   assert (Gathering);
 
   PAD_LOOP (Element);
@@ -1570,36 +1569,6 @@ DrawElementPinsAndPads (ElementTypePtr Element)
     DrawPin (pin);
   }
   END_LOOP;
-=======
-  if (Gathering)
-    {
-      PAD_LOOP (Element);
-      {
-        if (doing_pinout || doing_assy || FRONT (pad) || PCB->InvisibleObjectsOn)
-          draw_pad (pad);
-      }
-      END_LOOP;
-      PIN_LOOP (Element);
-      {
-        DrawPin (pin);
-      }
-      END_LOOP;
-    }
-  else
-    {
-      PAD_LOOP (Element);
-      {
-        if (doing_pinout || doing_assy || FRONT (pad) || PCB->InvisibleObjectsOn)
-          DrawPad (pad);
-      }
-      END_LOOP;
-      PIN_LOOP (Element);
-      {
-        draw_pin (pin, true);
-      }
-      END_LOOP;
-    }
->>>>>>> patched
 }
 
 /* ---------------------------------------------------------------------------
